chromedriver_autoinstaller/utils.py

```diff
@@ -225,6 +225,14 @@ def get_matched_chromedriver_version(chrome_version, no_ssl=False):
         for good_version in good_version_list["versions"]:
             if good_version["version"] == chrome_version:
                 return chrome_version
+        # get last major version
```

Comment:
I was a little unclear on what to expect from this comment.
```suggestion
        # Could not find an exact version match. 
        # Instead get the latest version of the driver matching the major version of the browser
```